pkg/services/smtp/smtp.go

```diff
@@ -59,15 +60,26 @@ func (service *Service) Send(message string, params *types.Params) error {
 	if params == nil {
 		params = &types.Params{}
 	}
-	client, err := getClientConnection(service.config.Host, service.config.Port)
+	client, err := getClientConnection(service.config.Host, service.config.Port, service.config.Encyption)
 	if err != nil {
 		return fail(FailGetSMTPClient, err)
 	}
 	return service.doSend(client, message, *params)
 }
 
-func getClientConnection(host string, port uint16) (*smtp.Client, error) {
-	conn, err := net.Dial("tcp", fmt.Sprintf("%s:%d", host, port))
+func getClientConnection(host string, port uint16, encryption encMethod) (*smtp.Client, error) {
+
+	var conn net.Conn
+	var err error
+
+	if useImplicitTLS(encryption, port) {
+		conn, err = tls.Dial("tcp", fmt.Sprintf("%s:%d", host, port), &tls.Config{
+			ServerName: host,
+		})
```

Comment:
```suggestion
        url := fmt.Sprintf("%s:%d", host, port)
        tlsConfig := &tls.Config{ServerName: host}
        
		conn, err = tls.Dial("tcp", url, tlsConfig)
```

I know you use GoLand, but I assume most readers don't. 😄

Reply:
split out the Sprintf to `addr` variable. the tls-config is verbose enough IMO.